challenges/functions.js

```diff
@@ -37,6 +37,7 @@ function myFunction() {
   function nestedFunction() {
     console.log(internal);
   };
-  nestedFunction();
+  return nestedFunction;
 }
-myFunction();
+const closureFunction = myFunction();
+closureFunction();
```

Comment:
Both versions of the code seem to work. This exercise deals in lexical scoping. Do you think the changes proposed make the rules of visibility clearer? Is this what you refer to as the "closure that wasn't"?

Reply:
So........ The problem is a bit hard to explain. Yes the function works and yes it can access the lexical scope but it's not a closure like it states above.
It can access it because was declared first. not because its a closure.

Reply:
All functions in JS are closures! They all "remember" the bindings in the environment they were defined in, even when that environment has gone away (returned). Your version makes the "even" part super explicit. Let's come up with another name for "closureFunction"! This way we don't give ideas that closures have to be built "just so". Something that makes the point that "closureFunction" points exactly to "nestedFunction". Can you think of anything? Something like "nestedFunctionAlso"? Mh. I don't like that...